packages/manager/modules/pci/src/projects/project/additional-ips/order/order.controller.js

```diff
@@ -246,7 +245,8 @@ export default class AdditionalIpController {
     } = this.publicCloudCatalog?.addons.find(
       (addon) => addon.product === FLOATING_IP_PLAN,
     );
-    const failoverIpPrice = `0${this.user.currency.symbol}`;
+
+    const failoverIpPrice = `${this.ipFailoverFormattedCatalog.plans[0].details.pricings.default[0].price.value}${this.user.currency.symbol}`;
```

Comment:
Is this constant used at any time without the symbol? If not, you can use .text instead of value + symbol

Reply:
I see it is used. 
` const failoverIpPrice = `${this.ipFailoverFormattedCatalog.plans[0].details.pricings.default[0].price.value}${this.user.currency.symbol}`;
    this.ipTypes = map(IP_TYPE_ENUM, (type) => {
      return {
        name: type,
        label: this.$translate.instant(`pci_additional_ip_${type}`),
        description: this.$translate.instant(
          `pci_additional_ip_${type}_description`,
        ),
        price:
          type === IP_TYPE_ENUM.FLOATING ? floatingIpPrice : **failoverIpPrice**,
      };
    });
    [this.selectedIpType] = this.ipTypes;
    this.initCountries();`

Reply:
Basically this symbol is the currency of the sub ($,€) for example

Reply:
It is used below. Earlier, it was used without defining. but now it is proper.

Reply:
I know. But this.ipFailoverFormattedCatalog.plans[0].details.pricings.default[0].price.text could replace
${this.ipFailoverFormattedCatalog.plans[0].details.pricings.default[0].price.value} + ${this.user.currency.symbol}

Because the text variable already contains both.